### ISSUE TITLE
[question] How to remove volumes with stale allocation state?

### ISSUE THREAD

**Issue by mkrueger-sabio** [hashicorp/nomad#9230]
I still have problems removing volumes where an allocation is associated with it but the allocation does not exist anymore. 

```
~ ⌚ 13:55:22
$ nomad volume deregister -force test-volume
Are you sure you want to force deregister volume "test-volume"? [y/N] y
Error deregistering volume: Unexpected response code: 500 (rpc error: volume in use: test-volume)

~ ⌚ 13:55:26
$ nomad volume status test-volume           
ID                   = test-volume
Name                 = test-volume
External ID          = labs-maestro
Plugin ID            = csi.serviceware.gluster.labs2
Provider             = csi.serviceware.gluster
Version              = 0.0.10
Schedulable          = false
Controllers Healthy  = 0
Controllers Expected = 8
Nodes Healthy        = 0
Nodes Expected       = 8
Access Mode          = multi-node-multi-writer
Attachment Mode      = file-system
Mount Options        = <none>
Namespace            = default

Allocations
ID        Node ID   Task Group  Version  Desired  Status   Created     Modified
56d0ec9b  7d72f8a4  httpd       2        run      running  3mo16d ago  3mo16d ago

~ ⌚ 13:55:34
$ nomad alloc status 56d0ec9b               
No allocation(s) with prefix or id "56d0ec9b" found
```

How can I remove them?

**Comment by tgross** [comment]
> I still 

@mkrueger-sabio what version of Nomad are you on?

**Comment by tgross** [comment]
WIP PR is https://github.com/hashicorp/nomad/pull/9239 and I'm looking to ship this in the 1.0 final.

**Comment by tgross** [comment]
Closed by #9239
